docs/getting-started.rst

```diff
@@ -1,56 +1,94 @@
-Getting Started
```

Comment:
Please explain the removal of all this content. This does not look correct. Thanks!